source/css/sass/patterns/molecules/article-type.scss

```diff
@@ -0,0 +1,20 @@
+@import "../../mixins/types";
+@import "../../variables/baselinegrid";
+@import "../../variables/color";
+@import "../../variables/font";
+@import "../../mixins/spacing";
+
+.article__type {
+  @include small-spacing();
+
+  a {
+    @include small-typography();
+    @include _label-typography($color-text-secondary, $uppercase: true);
```

Comment:
`_` indicates something private, we can't use it from outside.

Reply:
Removed _

Reply:
We shouldn't make that part of the public API, instead use `label-content-typography`.